documentation/tags.md

```diff
@@ -13,6 +13,7 @@ Anchor
 Android
 Angular
 Animation
+ANN
```

Comment:
```suggestion
```
An ANN tag is duplicative with "Neural Networks."

Reply:
done